app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -61,41 +71,34 @@ keywords:
   - Settlement Scheme
 ---
 $applications-open$
+  
+If you are considering coming to England to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.  
```

Comment:
@laurahermionetennant92 think it needs to be clear here this for teachers. Realise it says it in the box at the top but people may not read that or skim through it.

Reply:
If you're a teacher considering...